### PR TITLE
fixed image and create struc test

### PR DESCRIPTION
**Pull request by nhatdaoquang25**

### CODE REVIEW

File: lib/src/widgets/custom_card.dart [hunk 1]
```diff
@@ -48,7 +48,7 @@ class CustomCard extends StatelessWidget {
                 children: [
                   Padding(
                     padding: const EdgeInsets.all(12.0),
-                    child: Image.network(
+                    child: Image.asset(
```
Comment (HaiNguyenPV12) [inline lib/src/widgets/custom_card.dart]: Is there a particular reason to change where the image is fetched from?

Comment (nhatdaoquang25) [reply lib/src/widgets/custom_card.dart]: When I tested a widget,  file test war error. It's said You must have httpClient . So, I change it to test the screens.

Comment (nhatdaoquang25) [reply lib/src/widgets/custom_card.dart]: Can I change it or not ?

Comment (HaiNguyenPV12) [reply lib/src/widgets/custom_card.dart]: If that is the case then you cannot. Look up how to test widgets that rely on network calls 😉 .

Additionally, This widget may break if the API dies or the device fails to fetch the content. Look up how to remedy that.
```suggestion
                    child: Image.network(
```

**Review by HaiNguyenPV12 — DISMISSED** [review]
Change the current flow of image handling

File: lib/src/widgets/custom_card.dart (hunk 1 above)
Comment (nhatdaoquang25) [reply lib/src/widgets/custom_card.dart]: Yah, I will change this code like before. After that, I will solve problem for the test.

Comment (nhatdaoquang25) [reply lib/src/widgets/custom_card.dart]: I have just committed this file. You can review that.

File: pubspec.yaml [hunk 2]
```diff
@@ -29,7 +29,8 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-
+  mockito: ^5.0.13
+  mocktail: ^0.1.4
```
Comment (thanhphamtrung) [inline pubspec.yaml]: mocktail is enough

Comment (nhatdaoquang25) [reply pubspec.yaml]: I will fixed it.

**Review by thanhphamtrung — APPROVED** [review]

**Pull request merged**